test/e2e/scheduler_test.go

```diff
@@ -307,8 +307,8 @@ func TestMetricsAccessible(t *testing.T) {
 		t.Fatalf("error creating route client for prometheus: %v", err)
 	}
 	for _, metric := range []string{
-		"scheduler_schedule_attempts_total", // returned by /metrics
-		"kube_pod_resource_request",         // returned by /metrics/resources
+		"scheduler_schedule_attempts_total",    // returned by /metrics
+		"kube_pod_container_resource_requests", // returned by /metrics/resources
```

Comment:
/hold 
The change is wrong based on https://github.com/openshift/kubernetes/blob/master/pkg/scheduler/metrics/resources/resources.go#L54